controllers/space/space_controller.go

```diff
@@ -269,6 +296,41 @@ func (r *Reconciler) newNSTemplateSet(namespace string, name string, tmplTier *t
 	return nsTmplSet
 }
 
+func (r *Reconciler) newNSTemplateSetSpec(space *toolchainv1alpha1.Space, tmplTier *toolchainv1alpha1.NSTemplateTier) toolchainv1alpha1.NSTemplateSetSpec {
```

Comment:
the same function should be used also when the NSTemplateSet is being created
https://github.com/codeready-toolchain/host-operator/blob/31e0012331f43c216abf396c6592f3e3e1c4c630/controllers/space/space_controller.go#L286-L297
so we can be sure that the spec is created in the same way.

Reply:
done in 4d0f300